frontend/pages/index.vue

```diff
@@ -1,27 +1,235 @@
-<script setup>
-// Demo für Routing zu lokalen Seiten und Sprachwechsel
-const { locale, setLocale } = useI18n();
-const localeRoute = useLocaleRoute();
+<script lang="ts" setup>
+import { useWindowScroll } from "@vueuse/core";
 
-// demo for data fetching
-const client = useSanctumClient();
+const { y } = useWindowScroll();
+const scrollToTopButtonIsVisible = ref();
+const scrollToTopOfPage = () => {
+  window.scrollTo({ top: 0, behavior: "smooth" });
+};
```

Comment:
```suggestion
const scrollToTopOfPage = () => {
  y.value = 0;
};
```
das geht auch mit diesem Composable